test/hotspot/jtreg/compiler/vectorization/runner/VectorizationTestRunner.java

```diff
@@ -80,8 +80,6 @@ protected void run() {
         // To test vectorizability, invoke the IR test framework to check existence of
         // expected C2 IR node.
         TestFramework irTest = new TestFramework(klass);
-        // Add extra VM options to enable more auto-vectorization chances
-        irTest.addFlags("-XX:-OptimizeFill");
```

Comment:
Aha, you removed this too. Fair enough. But since the runner is currently requiring everything to be `flagless`, now I cannot actually force `-XX:-OptimizeFill` from the outside. And that means that potentially the tests are never actually run with `OptimizeFill` off, and we never actually can check the IR rules. We lose test coverage. That makes me a bit nervous.

Suggestion: if tests actually require the flag off to execute the IR rule, then we should have two scenarios, one where the flag is on, and one when it is off.

Reply:
Again, will cleanup this in JDK-8309697.

Reply:
It's already cleaned up in another patch. Will close it.